test/common/common/callback_impl_test.cc

```diff
@@ -53,5 +58,80 @@ TEST_F(CallbackManagerTest, DestroyManagerBeforeHandle) {
   handle.reset();
 }
 
+class ThreadSafeCallbackManagerTest : public testing::Test {
+public:
+  MOCK_METHOD(void, called, (int arg));
+};
+
+TEST_F(ThreadSafeCallbackManagerTest, All) {
```

Comment:
nit: we would appreciate short comments describing these new tests (as in drain_manager_impl_test.cc)

Reply:
Added!